docker-compose.yaml

```diff
@@ -2,7 +2,7 @@ version: '3'
 services:
   backend1:
     image: sensu/sensu:master
-    command: sensu-backend start --etcd-listen-client-urls http://0.0.0.0:2379 --etcd-name backend1 --etcd-advertise-client-urls http://backend1:2379 --etcd-initial-cluster backend1=http://backend1:2380,backend2=http://backend2:2380,backend3=http://backend3:2380 --etcd-initial-cluster-state new --etcd-initial-advertise-peer-urls http://backend1:2380 --state-dir /var/lib/sensu/sensu-backend/etcd1 --etcd-listen-peer-urls http://0.0.0.0:2380 --log-level info --debug
+    command: sensu-backend start --etcd-listen-client-urls http://0.0.0.0:2379 --etcd-name backend1 --etcd-advertise-client-urls http://backend1:2379 --etcd-initial-cluster backend1=http://backend1:2380,backend2=http://backend2:2380,backend3=http://backend3:2380 --etcd-initial-cluster-state new --etcd-initial-advertise-peer-urls http://backend1:2380 --state-dir /var/lib/sensu/sensu-backend/etcd1 --etcd-listen-peer-urls http://0.0.0.0:2380 --log-level debug --debug
```

Comment:
If that docker-compose file is aimed at developers, I'm fine with a debug log level. If it's aimed at operators, so they can quickly spin up a system to try out sensu or use it as a template to build their own, more complex docker-compose file, I would argue an info log level is probably more desirable.

Reply:
This is a really good point. It's definitely meant to be an example template for operators, but can be used for developers. When I was testing this feature in a clustered setting, I used this docker config, and actually thought there was a bug because `backend1` wasn't logging what I was expecting - turns out its because it had a different log level. I was hoping to mitigate the same confusing for someone else in the future.